src/components/CreateRules/UploadRulesSummary.vue

```diff
@@ -69,12 +64,36 @@ export default class UploadRulesSummary extends Mixins(DocumentMixin) {
 }
 
 .upload-rules-error-message {
-  padding-top: 1.25rem;
+  padding-top: 30px;
+  padding-bottom: 30px;
   padding-left: 1.25rem;
   color: $app-blue;
+
+  .error-text {
+    margin-right: 4px;
+  }
+}
+
+.upload-rules-success-message {
+  padding-top: 30px;
+  padding-bottom: 30px;
+  padding-left: 1.25rem;
+  color: $gray7;
+
+  .upload-success-chk {
+    margin-right: 9px;
+    font-size: 1.5rem;
+    color: #2e8540;
```

Comment:
Again, declare a named colour.

Reply:
added `$app-dk-green` to `theme.scss` and updated `.upload-success-chk` to use `$app-dk-green`